### ISSUE TITLE
Shrine - Textfields missing semantic label when active (2)

### ISSUE THREAD

**Issue by rami-a** [material-components/material-components-flutter-gallery#481]
## Acceptance Criteria
* Semantic label when active

**Comment by perclasson** [comment]
Can't reproduce this on iOS or Android. See screen reader recording below from iPhone:
[screen_reader_login.zip](https://github.com/material-components/material-components-flutter-gallery/files/4123420/screen_reader_login.zip)
